pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -1,6 +1,6 @@
-{ lib, stdenv, fetchurl, fetchpatch, python3Packages, zlib, pkg-config, glib, overrideSDK, buildPackages
-, pixman, vde2, alsa-lib, flex, pcre2
-, bison, lzo, snappy, libaio, libtasn1, gnutls, nettle, curl, dtc, ninja, meson, perl
+{ lib, stdenv, fetchurl, fetchpatch, python3Packages, zlib, pkg-config, glib, buildPackages
+, pixman, vde2, alsa-lib, flex
+, bison, lzo, snappy, libaio, libtasn1, gnutls, curl, dtc, ninja, meson, perl
```

Comment:
Removing unused arguments would ideally be a separate commit.

Reply:
not that it matters but whatever